### DISCUSSION TITLE
Media manager

### DISCUSSION THREAD

**Discussion by gsicvj** [tinacms/tinacms#3933] [Help] [answered]
Hi maintainers, I'm trying to figure out how to display images in production without assets.tina.io.
Gatsby project images are managed in static/ and sourced from public/ when live.

I am a bit lost and can't seem to find a solution after reading different discussions and documentation.

What I checked so far:
- https://github.com/tinacms/tinacms/discussions/3917
- https://tina.io/docs/reference/media/overview/

This is a project config:
```
build: {
  outputFolder: "admin",
  publicFolder: "static",
},
media: {
  tina: {
    mediaRoot: "images",
    publicFolder: "static",
  },
},
```

If mediaRoot is empty string, the images aren't displayed properly on the web.

At the start of forestry migration images weren't displayed in Tina UI.
I played around with different configuration setups, but in the end the synchronisation with Tina cloud solved that.

However, the things started to break.

Problems:
- Images are sometimes fetched from assets.tina.io in production
- Images are sometimes not saved properly to github

Questions:
- Is the synchronisation to tina cloud really necessary?
- How to fix synchronisation issues after config paths change and synchronisation button is slicked?

Note:
- The current project setup isn't optimal, and maybe we'll connect to external media provider in the future
- It took a while to notice this issues since our first changes were textual changes, so I don't know which setup resulted in certain behaviour.
- Any help or guidance would be helpful

**Comment by jamespohalloran** [comment, marked as answer]
> Images are sometimes not saved properly to github

Think you may have been running into this? https://github.com/tinacms/tinacms/issues/3892 
If so, it should be fixed in the latest version

> Images are sometimes fetched from assets.tina.io in production

Are you using Tina's data-fetching on your site? If not, you may just be running into the issue above.

> How to fix synchronisation issues after config paths change and synchronisation button is slicked?

Currently, if you change your directory for your synced media, you will need to start a new project. We're looking to add some dashboard functionality to make this a better experience. 

> Is the synchronisation to tina cloud really necessary?:

Yes we need to index your media so that we can properly display it in the Tina admin.

**Reply by gsicvj** [reply]
It looks like we indeed have been running into #3892, thank you for your answers.
